Pokemon Unity/Assets/Editor/CustomEventEditor.cs

```diff
@@ -385,9 +385,10 @@ private string GetEventDescription(SerializedProperty currentSEvent)
                 break;
 
             case CustomEventDetails.CustomEventType.ReceivePokemon:
-                eventDescription = "Receive a Lv. " + ints_Prop.GetArrayElementAtIndex(1).intValue + " \"";
-                PokemonDataOld pkd = PokemonDatabaseOld.getPokemon(ints_Prop.GetArrayElementAtIndex(0).intValue);
-                eventDescription += (pkd != null) ? pkd.getName() : "null";
+                eventDescription = "NOT CURRENTLY FUNCTIONAL AWAITING FIX";
```

Comment:
I find it extremely funny that you renamed description instead of making notes in a comment. Im glad you made attempts to fix so much!